pkg/step/feature/derivation.go

```diff
@@ -486,6 +487,23 @@ func setBackToIR(trainStmt *ir.TrainStmt, fcMap ColumnMap, columnTargets []strin
 	}
 }
 
+// setBackToLabel update label field meta
+func setBackToLabel(trainStmt *ir.TrainStmt, fmMap FieldDescMap) {
+	// NOTE: clustering model may not specify Label
+	labelName := trainStmt.Label.GetFieldDesc()[0].Name
+	if labelName == "" {
+		return
```

Comment:
I see. How about moving the comment `// NOTE: clustering model doesn't have label` to the end of this line?

Reply:
Done.